packages/account/src/Components/financial-details/financial-details.tsx

```diff
@@ -84,9 +87,27 @@ const FinancialDetails = (props: TFinancialDetails) => {
                                     height_offset='110px'
                                     is_disabled={isDesktop()}
                                 >
-                                    <Text as='p' color='prominent' size='xxs' className='trading-assessment__side-note'>
-                                        <Localize i18n_default_text='We collect information about your employment as part of our due diligence obligations, as required by anti-money laundering legislation.' />
-                                    </Text>
+                                    {is_eu_user && (
+                                        <div className='details-form__banner-container'>
+                                            <InlineNoteWithIcon
+                                                icon='IcAlertWarning'
+                                                message={localize(
```

Comment:
This message we can use Localize as message is of type ReactNode in InlineNoteWithIcon